### DISCUSSION TITLE
If you were to switch backends

### DISCUSSION THREAD

**Discussion by sumibi-yakitori** [17cupsofcoffee/tetra#284] [Ideas]
@17cupsofcoffee 
Currently, Tetra is using SDL2 as the backend for desktops, but do you have plans for the far future to switch to a different backend?

If so, what do you plan to use?

**Comment by sumibi-yakitori** [comment]
Oops, I overlooked the FAQ

I wanted to delete this discussion, but I don't seem to have the authority to do so, even if I delete this reply.

**Comment by 17cupsofcoffee** [comment]
That section of the FAQ hasn't been updated for a while, so I think it's still a fair question!

I would consider moving to `winit` someday for easier builds/packaging and WASM support, but SDL2 seems more stable/battle-tested at the moment, and `winit` is missing some things that I'd want (e.g. gamepad support). So I wouldn't expect the desktop backend to change in the short-term.

I may add additional windowing backends in the future, though (e.g. a web backend).
